src/Configuration/test/Config.FunctionalTests/ConfigurationTests.cs

```diff
@@ -463,9 +464,8 @@ public void OnLoadErrorCanIgnoreErrors()
             Assert.NotNull(provider);
         }
 
-        [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Linux, SkipReason = "File watching is flaky on non windows.")]
-        [OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "File watching is flaky on non windows.")]
+        [Fact]
+        [Flaky("File watching is flaky (particularly on non windows.", FlakyOn.All)]
```

Comment:
This is supposed to be a link to a GitHub issue tracking re-enabling it, or at least describing the details. If the test is permanently flaky, we should consider how valuable it actually is :)